projects/tizio.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: TIZIO – Transfer- und Innovationszentrum im Oberland 
+---
+
+We are part of the newly founded TIZIO (Transfer- und Innovationszentrum im Oberland) based in Bad Tölz. 
+
+
+![TIZIO Building](/assets/img/tizio.png)
+
+
+In this lab, we work on solutions related to:
+
+- Digitization of production
+- IT security & zero-trust environment
+- Embedded systems 
+
+We also investigate concepts such as WebAssembly for Embedded Systems. Here, an efficient in-line interpreter for WASM programms was developed that can execute the bytecode without having to convert it into an intermediate representation.
+
+Link to : [TIZIO](https://hm.edu/kooperationen_transfer/tizio/index.de.html)
+Link to : [TIZIO IoT Lab](https://hm.edu/kooperationen_transfer/tizio/labor_iot.de.html)
```

Comment:
better to reference this directly in the text. Also the "Link to :" will not look nicely, and both would be on the same line.